UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/ISalesCatalogueClient.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace UKHO.ExchangeSetService.Common.Helpers
+{
+    public interface ISalesCatalogueClient
+    {
+        public Task<HttpResponseMessage> CallSalesCatalogueServiceApi(HttpMethod method, string postBody, string authToken, string uri);
```

Comment:
PUT method will also have request body. Please can we consider to rename it accordingly.
```suggestion
        public Task<HttpResponseMessage> CallSalesCatalogueServiceApi(HttpMethod method, string requestBody, string authToken, string uri);
```

Reply:
Made changes related to property name in 6850 PBI PR